pkg/server/field_mask_test.go

```diff
@@ -0,0 +1,294 @@
+package server
```

Comment:
Since this PR focuses on updating a service using mask, would be good to see some integration tests for the GRPC service.

Reply:
They already exist, both in gRPC and REST